src/open_samus_returns_rando/files/levels/s000_surface.lua

```diff
@@ -93,6 +93,11 @@ function s000_surface.InitFromBlackboard()
     Game.SetPlayerInputEnabled(false, false)
     s000_surface.LaunchPlanetArrival()
   end
+  if not Blackboard.GetProp("DEFEATED_ENEMIES", "Ridley") and Game.GetEntity("LE_Item_Ridley") ~= nil then
+    Game.DisableEntity("LE_Item_Ridley")
+  else
+    Game.EnableEntity("LE_Item_Ridley")
+  end
```

Comment:
There was something weird about this check which I forgot about: Can move and surround both by the `Game.GetEntity("LE_Item_Ridley") ~= nil` check? It kinda scares me to call `EnableEntity` on a non existing one.